File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.py
class Solution:
    def replaceElements(self, arr: List[int]) -> List[int]:
        mx=max(arr)
        for i in range (len(arr)-1):
            
            if(arr[i]!=mx):
                arr[i]=mx
            else : 
                mx=max(arr[i+1:])
                #print(arr[i:])
                arr[i]=mx
        arr[len(arr)-1]=-1
        return arr



        
        
        

File: Data Structures & Algorithms/climbing-stairs/submission-0.cpp
class Solution {
public:
    int f(int n,int i){
        if(i==n) return 1;
        else if(i>n) return 0;
        else{
            return f(n,i+1)+f(n,i+2);
        }
    }
    int climbStairs(int n) {
        return f(n,0);
    }
};
